src/components/Card.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./Card.css";
+import { Card as CardComp, Button } from "react-bootstrap";
+import { BsFillTrashFill } from "react-icons/bs";
+
+const Card = (props) => {
+  const onClickUpdateCard = () => {
+    const updateCard = {
+      ...props,
+      likesCount: props.likesCount + 1,
+    };
+    props.onUpdateLike(updateCard);
+  };
+  const onClickRemoveCard = () => {
+    const updateCard = {
+      ...props,
+      status: !props.status,
+    };
+    props.onRemove(updateCard);
+  };
```

Comment:
Tiny style nitpick: in most coding languages it is the best practice to use a single blank line to visually separate functions so it is very easy to see at a glance where one stops and another starts.